tests/FondOfSpryker/Yves/EnhancedEcommerceCheckoutConnector/Expander/PaymentAddressExpanderTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace FondOfSpryker\Yves\EnhancedEcommerceCheckoutConnector\Expander;
+
+use Codeception\Test\Unit;
+use FondOfSpryker\Yves\EnhancedEcommerceCheckoutConnector\Dependency\EnhancedEcommerceCheckoutConnectorToCartClientInterface;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Shared\Money\Dependency\Plugin\MoneyPluginInterface;
+
+class PaymentAddressExpanderTest extends Unit
+{
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|\FondOfSpryker\Yves\EnhancedEcommerceCheckoutConnector\Dependency\EnhancedEcommerceCheckoutConnectorToCartClientInterface
+     */
+    protected $cartClientMock;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|\Spryker\Shared\Money\Dependency\Plugin\MoneyPluginInterface
+     */
+    protected $moneyPluginMock;
+
+    /**
+     * @var \FondOfSpryker\Yves\EnhancedEcommerceExtension\Dependency\EnhancedEcommerceDataLayerExpanderInterface
+     */
+    protected $expander;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|\Generated\Shared\Transfer\QuoteTransfer
+     */
+    protected $quoteTransferMock;
+
+    /**
+     * @return void
+     */
+    protected function _before(): void
+    {
+        $this->cartClientMock = $this->getMockBuilder(EnhancedEcommerceCheckoutConnectorToCartClientInterface::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->moneyPluginMock = $this->getMockBuilder(MoneyPluginInterface::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->quoteTransferMock = $this->getMockBuilder(QuoteTransfer::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->expander = new PaymentAddressExpander($this->cartClientMock, $this->moneyPluginMock);
+    }
+
+    /**
+     * @return void
+     */
+    public function testExpand(): void
+    {
+        $this->cartClientMock->expects($this->atLeastOnce())
+            ->method('getQuote')
+            ->willReturn($this->quoteTransferMock);
+
+        $this->quoteTransferMock->expects($this->atLeastOnce())
+            ->method('getItems')
+            ->willReturn([]);
+
+        $this->expander->expand('pageType', [], []);
+    }
+}
```

Comment:
Shouldn't this test that the expand function returns the array with expected values?